web/source/keymanweb.js

```diff
@@ -3681,21 +3722,30 @@
     } else {
       if(Pelem.tagName.toLowerCase() != 'iframe') {
         if(keymanweb.isKMWInput(Pelem)) {
-          var x = keymanweb.setupTouchElement(Pelem);
-
-          if(x) {
-            // if needed, let the timeout variable be: keymanweb.touchInputAddedTimer
-            window.setTimeout(function() {
-              keymanweb.updateInput(x);
-            }, 1);
-          }
+          keymanweb.setupTouchElement(Pelem);
```

Comment:
Could we refactor `setupNontouchElement` and `setupTouchElement` into a single parent function call and move both the `iframe` test and the `isKMWInput` into the parent function?  This simplifies both places where these functions are called. Then also rename `setupNontouchElement` to something like `setupNonKMWTouchElement`?

line:3722:
````
  else {
    keymanweb.setupTouchElement(Pelem);
  }
````

line:860:
````
      for(var n=0;n<ipList.length;n++) { 
        keymanweb.setupTouchElement(ipList[n]);
      }
````
....
line:883:
````
    keymanweb.setupTouchElement = function(Pelem)
    {
      // We don't support iframe for touch devices at present
      if(Pelem.tagName.toLowerCase() == 'iframe') {
        return null;
      }

      // The specified element must be validated as a touch element to proceed.
      if(!keymanweb.isKMWInput(Pelem)) {
        return keymanweb.setupNonKMWTouchElement(Pelem);
      }
      ....
````